packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable func-style */
+import { PlusIcon } from '@heroicons/react/24/solid';
+import type { Meta, StoryFn } from '@storybook/react';
+
+import { Button } from '.';
+
+const Icon = PlusIcon;
+
+export default {
+    title: 'Buttons/Button',
+    component: Button,
+} as Meta<typeof Button>;
+
+const ButtonTemplateWithChild: StoryFn<typeof Button> = (args) => <Button {...args}>Button</Button>;
+const ButtonTemplateWithoutChild: StoryFn<typeof Button> = (args) => <Button {...args} />;
+
+export const Primary = ButtonTemplateWithChild.bind({});
+Primary.args = {
+    type: 'button',
+    variation: 'primary',
+};
+
+export const Secondary = ButtonTemplateWithChild.bind({});
+Secondary.args = {
+    ...Primary.args,
+    variation: 'secondary',
+};
+
+export const PrimaryWithIconLeft = ButtonTemplateWithChild.bind({});
+PrimaryWithIconLeft.args = {
+    ...Primary.args,
+    icon: Icon,
+};
+
+export const PrimaryWithIconRight = ButtonTemplateWithChild.bind({});
+PrimaryWithIconRight.args = {
+    ...Primary.args,
+    icon: Icon,
+    iconPlacement: 'right',
+};
+
+export const SecondaryWithIconLeft = ButtonTemplateWithChild.bind({});
+SecondaryWithIconLeft.args = {
+    ...Secondary.args,
+    icon: Icon,
+    iconPlacement: 'left',
+};
+
+export const SquareButton = ButtonTemplateWithoutChild.bind({});
```

Comment:
Square button is not exactly square 😉 

<img width="380" alt="image" src="https://github.com/prezly/theme-kit-js/assets/1641218/2210a5c9-edb9-43bd-b7c2-0b31bd6152e3">

Reply:
Also, I think the line height for the button is weird, I was expecting round values for text line-height, at least for buttons.
<img width="272" alt="image" src="https://github.com/prezly/theme-kit-js/assets/1641218/a9db7eba-f6f2-4c08-932c-24fe46ab5ebe">

Reply:
Updated https://github.com/prezly/theme-kit-js/pull/404/commits/f4b350576eda805db5368a8be746d725d349af4c